pkg/eve/runme.sh

```diff
@@ -21,7 +21,7 @@ dump() {
             INAME="$INAME.qcow2"
             ONAME="$ONAME.qcow2"
             ;;
-     parallels) qemu-img convert -c -f raw -O parallels "$INAME" "$INAME.parallels"
+     parallels) qemu-img convert -f raw -O parallels "$INAME" "$INAME.parallels"
```

Comment:
why are we dropping copmression?

Reply:
This is a limitation of qemu-img. Only the formats "qcow" and "qcow2" support compression. 

https://github.com/qemu/qemu/blob/c99fa56b95a72f6debd50a280561895d078ae020/docs/tools/qemu-img.rst